guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -37,66 +37,117 @@ The exported archive file contains the following data:
-endif::[]
-
 .Prerequisites
 
 To export a Content View, ensure that the {ProjectServer} where you want to export meets the following conditions:
 
 * Ensure that the export directory has free storage space to accommodate the export.
-* Ensure that the `/var/lib/pulp/` directory has free storage space equivalent to the size of the repositories being exported for temporary files created during the export process.
-* Ensure that the `/var/cache/pulp` directory has free storage space equivalent to twice of the size of the repository being exported for temporary files created during the export process.
+* Ensure that the `/var/lib/pulp/exports` directory has free storage space equivalent to the size of the repositories being exported for temporary files created during the export process.
 * Ensure that you set download policy to *Immediate* for all repositories within the Content View you export.
 For more information, see xref:Importing_Content-Configuring_Download_Policies[].
-* Ensure that you clear the *Mirror on Sync* check box for the repositories that you import on the repository settings page.
 * Ensure that you synchronize Products that you export to the required date.
 
 .To Export a Content View Version:
 
-. List Content Views to determine the ID of a Content View version you want to export:
+. List versions of the Content View that are available for export:
 +
 [subs="+quotes"]
 ----
-# hammer content-view version list \
---organization "_Default Organization_"
+
+# hammer content-view version list --organization=export-org --content-view=view
+---|----------|---------|-------------|-----------------------
+ID | NAME     | VERSION | DESCRIPTION | LIFECYCLE ENVIRONMENTS
+---|----------|---------|-------------|-----------------------
+5  | view 3.0 | 3.0     |             | Library
+4  | view 2.0 | 2.0     |             |
+3  | view 1.0 | 1.0     |             |
+---|----------|---------|-------------|----------------------
+
 ----
 
-. Export the version of a Content View.
-Specify the directory where to store the export with the `--export-dir` option and the ID of the Content View version that you export with the `--id` option.
-The `pulp_export_destination` setting does not work for this procedure.
-+
+.Export the version of a Content View.
+Get the version number of desired version. In the case below we are interested in fully exporting version `1.0`.
+
 [options="nowrap" subs="+quotes"]
 ----
-# hammer content-view version export --export-dir _export_directory_ \
---id _content_view_version_ID_
+# hammer content-export complete version --content-view=view --version=1.0 --organization=export-21527
+[.....................................................................................................................................................................................................................................] [100%]
+Generated /var/lib/pulp/exports/export-21527/view/1.0/2021-02-25T18-59-26-00-00/metadata.json
```

Comment:
```suggestion

```
We don't expose the output in docs unless it is complex and vital to work with for the next step.